R/dependencies.R

```diff
@@ -1,4 +1,7 @@
 spark_dependencies <- function(spark_version, scala_version, ...) {
+  hail_exists <- system.file("java/hail-all-spark.jar", package = "sparkhail")
+  if (hail_exists == "") stop("Hail is uninstalled. Please use hail_install().")
```

Comment:
Rather than triggering an error, I would rather call `hail_install()` here when hail is not installede... extensions already require internet access to download packages from http://spark-packages.org, so downloading this automatically seems equivalent in this case.